lib/http.c

```diff
@@ -1092,6 +1095,283 @@ static bool http_should_fail(struct Curl_easy *data, int httpcode)
   return data->state.authproblem;
 }
 
+CURLcode Curl_http_follow(struct Curl_easy *data, const char *newurl,
```

Comment:
Can this be turned into a static function now? (and if so, drop the `Curl_` prefix)

Reply:
It is also used in RTSP and WS protocol handlers.

Reply:
Oops, I missed that. Case closed then.